src/components/Bookstore.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
```

Comment:
- [ ] Kindly check your source when importing `Routes` am getting this error while trying to run your project 
![image](https://user-images.githubusercontent.com/61408860/173434566-cd487ae0-06f6-4418-8de9-75cdb8ccb567.png)